lang/LangSource/PyrObject.cpp

```diff
@@ -181,7 +181,10 @@ void initSymbols()
 {
 	s_new = getsym("new");
 	s_ref = getsym("Ref");
-	s_none = getsym("none");
+	s_none = getsym("__none");
+	/* Dummy symbol for null superclass or primitive. Prefixed with
+	 * `__` to avoid collisions with possible method names.
```

Comment:
> I suppose that the double `_`-prefix (instead of a single one) is needed because single prefix `_` is reserved for primitive calls?

Technically it's not necessary, since primitives begin with _[A-Z], but this way it's clear that it's part of neither namespace.